app/scripts/directives/overview/serviceInstanceRow.js

```diff
@@ -32,12 +33,13 @@
     var serviceInstanceFailedMessage = $filter('serviceInstanceFailedMessage');
     var truncate = $filter('truncate');
 
-    _.extend(row, ListRowUtils.ui);
+    _.extend(row, ListRowUtils.ui)
```

Comment:
should this semi colon be missing?

Reply:
yeah not sure why that crept in. will update